Toolset/palettes/script editor/behaviors/revseeditorbehavior.livecodescript

```diff
@@ -951,7 +951,7 @@ command scriptComment
    # OK-2009-03-03 : Bug 7537 - Reformat the handler after commenting, this may be flawed,
    # but the proper solution is rather time-consuming.
```

Comment:
@montegoulding After mulling it over I think that makes sense anyway, as there might be cases where commenting or uncommenting over the boundaries of two handlers affects the indenting later on.